[2024-02-08T08:53:04.870+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 6_postgres.create_postgres_table manual__2024-02-08T08:53:03.580229+00:00 [queued]>
[2024-02-08T08:53:04.895+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 6_postgres.create_postgres_table manual__2024-02-08T08:53:03.580229+00:00 [queued]>
[2024-02-08T08:53:04.898+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-08T08:53:04.961+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-02-08 08:53:03.580229+00:00
[2024-02-08T08:53:04.972+0000] {standard_task_runner.py:60} INFO - Started process 7298 to run task
[2024-02-08T08:53:04.979+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '6_postgres', 'create_postgres_table', 'manual__2024-02-08T08:53:03.580229+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/6_postgres.py', '--cfg-path', '/tmp/tmprc83jl5i']
[2024-02-08T08:53:04.984+0000] {standard_task_runner.py:88} INFO - Job 82: Subtask create_postgres_table
[2024-02-08T08:53:05.133+0000] {task_command.py:423} INFO - Running <TaskInstance: 6_postgres.create_postgres_table manual__2024-02-08T08:53:03.580229+00:00 [running]> on host 8c35f9d61d9d
[2024-02-08T08:53:05.389+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cardthp' AIRFLOW_CTX_DAG_ID='6_postgres' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-08T08:53:03.580229+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-08T08:53:03.580229+00:00'
[2024-02-08T08:53:05.393+0000] {sql.py:276} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2024-02-08T08:53:05.414+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:53:05.453+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-08T08:53:05.492+0000] {sql.py:450} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2024-02-08T08:53:05.589+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=6_postgres, task_id=create_postgres_table, execution_date=20240208T085303, start_date=20240208T085304, end_date=20240208T085305
[2024-02-08T08:53:05.687+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-08T08:53:05.915+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
